.dockerignore

```diff
@@ -11,3 +11,4 @@ charts
 *.md
 *.txt
 *.sh
+test/e2e/.docker-build
```

Comment:
Not sure if we want to get aggressive with removing files but here's a few more:
```
*.stories.*
js-packages/mobile-app
docs/
.storybook/
.github/
extension/ # I think it's not being used by our Dockerfile
```